### DISCUSSION TITLE
noob question RE: Putty with Pico (8080 replica assembly step 8)

### DISCUSSION THREAD

**Discussion by gohills** [thehighnibble/thehighnibble.github.io#77] [Q&A]
Last time I used a serial connection was on an Apple //e.  Apps were single function and the only thing I had to configure was baud rate, 8N1, XON/OFF etc.  Simple. 

Using Windows with Putty and USB cable directly into the Pico.  VCP drivers are installed and seem to be working because when I plug in the Pico, COMM 5 pops up on the list of Windows devices ... windows sees the pico (probably good enough for step 8).  In Windows I configure COMM 5 for 115200 8N1.  In Putty I configure a Serial connection on COMM 5 with 115200 8N1.  Press the Open button and all I get is a 'beep'.  No terminal session opens up.  What am I missing?

Alternatively, using iTerm2 on the Mac.  Drivers installed, etc but I cannot even see how to configure a serial connection with iTerm2.  Seems like a bigger reach for me unless of course I am missing something that makes it REAL easy.

Hate asking for help with software tools but I'm getting frustrated.   TIA!

**Comment by dmcnaugh** [comment]
Note, that if you have the Web Desktop open then you wont get anything on the USB/Serial port.

**Reply by gohills** [reply]
Had not considered this because after all, I am only on step 8.  But ..... since you mentioned it, I decided to try and connect to the emulator via wifi and it immediately  popped up.  So I know the microSD and pico soldering is OK.  I am less clear about the flash memory since library images cannot be dragged onto the floppy drives.  For now, I am moving forward with soldering and hoping emulation will improve when I update the microSD.  I am still unable to connect using Putty.... all I get is a "bell" from Windows even though the VCP device is present on COMM5.

**Reply by dmcnaugh** [reply]
The COM port settings should be 115200 baud 8N1 as you've said, but check that they are set this way in Putty. You also need to set NO flow control. Putty often has this set on by default.

**Reply by kellert001** [reply]
I am having the same problem. I can not connect to eh ESP32-PICO via putty( shows COM3). I have tried on three different computers, verified the settings, putty just beeps when I try to connect.

**Comment by kellert001** [comment]
The problem I had with putty.  If you start putty, get set serial port setup, plug in the USB cable, wait for the host computer to recognize the device then open the putty terminal.  You should see the ESP32 boot up messages.  It was easier with a Linux box and the tio program (simple TTY terminal)
